pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -76,7 +76,7 @@
       <Version>1.0.2</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>1.0.12</Version>
+      <Version>1.0.13</Version>
```

Comment:
I think there will be a merge conflict here after I merge auto update PR

Reply:
You need to update Microsoft.NETCore.DotNetHostPolicy 1.0.0 dependency to 1.0.11 here as it always gets built for 1.0.0

Reply:
Updated